src/app/list-routine/list-routine.component.html

```diff
@@ -0,0 +1,15 @@
+<div id="listRoutineTopBar">   
```

Comment:
You can use 
`<input
    type="text"
    placeholder="Search in your routines"
    (input)="updateArticles($event.target.value)"
  >`